public/admin/config.yml

```diff
@@ -0,0 +1,368 @@
+backend:
+  name: git-gateway
+local_backend: true
```

Comment:
It's a temporary change so you will be able to run the admin panel using your local file system. I will move this line back to the previous state before merging this PR

Reply:
I have a couple of questions 💭 :

1. Is development mode SSR? - I'm asking this question because the page in development mode feels a bit slow locally. On the other hand, the admin panel is much faster compared to the previous version.
2. Does the production deployment have to be SSR or we still can go with SSG? 
3. Have you checked what changes do we have to do on Netlify in order for this deployment to work? For sure the build command but anything else? 
4. Do we need to deploy the server separately?

In general, this PR looks good to me and looks very similar to the previous PR with icons #184 so it kinda blurs which changes are which. Maybe we can merge the previous PR first and then get back to this one? Because it is unnecessarily big at the moment and it is hard for me to focus on the key changes and review them. 🙏 

In general, it looks good to me but I would like to review it in more detail without the model update part 😄
If you feel like changing to Next.js solves the problems that we have then I'm with you, I just have a small sentiment to the Gatsby version that's why I may be a bit skeptical.  🛩️ 🌍

Reply:
1. Yes, in development it's SSR and that's the reason why some actions (especially moving between pages) are slower.
2. When you build the website it's all SSG and there are no performance issues you had in the development mode.
3. Only the build command. I checked it in the docs for Netlify CMS.
4. No it will be still a one-click-deploy app.

I agree that it would be great to merge the #184 PR first as it contains all model changes you can see in this PR as well. Would it be a problem for you if I merge that PR without fixing the case of the optional icons?

Reply:
No problem, go ahead with the merge 👍 
I hope that CodersCard community will not be pulling it in the meantime 🤫

Reply:
Also thanks for responding, all answers are great news 🎉